src/components/Common/TableOld/TableHead/TableHead.js

```diff
@@ -229,7 +226,20 @@ export const TableHead = observer(
           });
           return orderedColumns;
         }, [columns]);
-  
+        
+        useEffect(() => {
+          ref.current?.addEventListener("mousedown", (event) => {
```

Comment:
why not just add this as `onMouseDown` direct handler?

Reply:
onMouseDown does not seem to be available here, am I missing something?